### DISCUSSION TITLE
In the new 'for you' feed, add a way to hide updates where the only reason is that I starred the repo

### DISCUSSION THREAD

**Discussion by LB--** [community/community#65779] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I wanted to try and clean up my 'for you' feed in the new beta feed layout, and I noticed the filter options. I unticked Stars and clicked apply, and the feed reloaded, and I expected it to remove repositories where the only reason for them being in my feed is that I starred them. However, the feed did not change at all after reloading. If this isn't what the Stars filter is for, then maybe a new filter should be added?
![image](https://github.com/community/community/assets/1296838/560bc857-997a-4550-9d99-d6a2984e5323)
![image](https://github.com/community/community/assets/1296838/fabe1eb9-8897-4bb0-bba1-c4dc088a92de)
![image](https://github.com/community/community/assets/1296838/efec8639-da02-4947-b677-ee79f3528f5f)
I notice the "See less activity like this" option, but I am scared to click it, because I do want to receive that _type_ of update, just not from repositories where the only interaction I've had with the repo is that I starred it, and I'm not sure which of those two different things is being referred to here. I also do not want to un-star the repository.

**Comment by DanielFelips-DF** [comment]
Mcilra
